tests/test_free_energy.py

```diff
@@ -317,14 +317,15 @@ def test_functional():
     box = np.eye(3) * 100
     lam = 0.5
 
-    for precision in [np.float32, np.float64]:
+    tol_at_precision = {np.float32: 2.5e-10, np.float64: 1e-10}
+    for precision, tol in tol_at_precision.items():
```

Comment:
Surprised this caused an issue locally but not on the test runner... Regardless, it's a good idea to have separate tolerances for the different precision modes!